apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/PermissionValidator.java

```diff
@@ -102,6 +102,12 @@ public boolean hasCreateClusterPermission(String appId) {
         appId);
   }
 
+  public boolean hasManageAccessKeyPermission(String appId) {
```

Comment:
how about just use `isAppAdmin` so that we don't need to a new permission type?

Reply:
Yeah, `isAppAdmin` is what I want! 
I'll will polish this logic code.

Reply:
`com.ctrip.framework.apollo.portal.component.PermissionValidator#isAppAdmin`
```java
public boolean isAppAdmin(String appId) {
  return isSuperAdmin() || hasAssignRolePermission(appId);
}
```
But I found `isAppAdmin` means whether it has `AssignRole` permission. We should aware that `AssignRole` permission has two meanings after we use it for `ManageAccessKey` permission.

Reply:
> But I found isAppAdmin means whether it has AssignRole permission. We should aware that AssignRole permission has two meanings after we use it for ManageAccessKey permission.

That should be OK.